SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -132,21 +132,9 @@ - (instancetype)initWithMessage:(TSOutgoingMessage *)message
 
 - (nullable NSError *)checkForPreconditionError
 {
-    for (NSOperation *dependency in self.dependencies) {
```

Comment:
I moved this "fail if a dependency failed" logic into the base OWSOperation class so we could use it elsewhere.